src/smjsindustry/finance/processor_config.py

```diff
@@ -224,7 +224,7 @@ class KMedoidsSummarizerConfig(FinanceProcessorConfig):
     k-medoids minimizes the sum of dissimilarities between vectors in a cluster
     and one of the vectors designated as the representative of that cluster;
     the representative vectors are called medoids. The collection of medoids
-    and the m sentences in the document closest to the cluster medoids is
+    and the m sentences in the document closest to the cluster medoids are
```

Comment:
and the m sentences in the document corresponding to the cluster medoids

Reply:
Will fix it in a follow-up PR.